### DISCUSSION TITLE
face swap error!

### DISCUSSION THREAD

**Discussion by arnabing** [lllyasviel/Fooocus#2417] [General]
Amazing job to the devs to make this project in G collab

Anyone else getting error with face swap, like this?:
"[Fooocus] Image processing ...
Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /content/Fooocus/models/controlnet/detection_Resnet50_Final.pth

100% 104M/104M [00:00<00:00, 276MB/s] 
Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.2.2/parsing_parsenet.pth" to /content/Fooocus/models/controlnet/parsing_parsenet.pth

100% 81.4M/81.4M [00:01<00:00, 57.6MB/s]
Detected 1 faces
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
^C"

**Comment by mashb1t** [comment]
=> https://github.com/lllyasviel/Fooocus/pull/1710 for additional parameters allowing better usage of resources in Colab

**Reply by arnabing** [reply]
Thank you! Switching to High Vram worked.
